### ISSUE TITLE
Linux installation error “nixbld”

### ISSUE THREAD

**Issue by sudomonzter** [input-output-hk/daedalus#2612]
Upon a fresh installation on Linux. I am being thrown error logs saying something on the lines of: 

(unfortunately I didn’t manage to capture the entire error log)

Partial error code:

warning: the group 'nixbld' specified in 'build-users-group' does not exist / warning: the group 'nixbld' specified in 'build-users-group' does not exist reading the Nix store... 

I was told to add additional codes in the terminal to solve this issue/ error only after the display of the error codes but I would like to get official directions from iohk and the Daedalus team as I want to install Daedalus from scratch the proper way.

**Comment by nikolaglumac** [comment]
@sudomonzter please provide more info about your system. Which Linux are you using? What version? etc...
Please be mindful of the supported Linux versions:  https://iohk.zendesk.com/hc/en-us/articles/360010496553-Daedalus-System-Requirements

@dmitrii-gaico please assist @sudomonzter here. Thanks!

**Comment by nikolaglumac** [comment]
@sudomonzter any news?

**Comment by dmitrii-gaico** [comment]
Closing the ticket as additional feedback required for further investigations. @sudomonzter please open a new ticket if you still facing the issues so we can resume the research.
